include/circt/Dialect/Calyx/CalyxPrimitives.td

```diff
@@ -93,3 +93,52 @@ def MemoryOp : CalyxPrimitive<"memory", [
     )>
   ];
 }
+
+class CalyxLibraryOp<string mnemonic, list<OpTrait> traits = []> :
+  CalyxPrimitive<"lib_" # mnemonic, traits> {
```

Comment:
Minor comment: The Calyx mnemonic is using the `std_` prefix. Up to you if you want match the Calyx naming convention or not.

Reply:
Seems reasonable to me - will fix.